### DISCUSSION TITLE
IIP-11: Increasing the cost of 51% attack

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-docs#143] [IIPs]
Increase the cost of 51% attack by disregarding votes from identities with stake amounts below a certain threshold. This threshold is set as a percentage of the median stake of top 100 accounts.

Read more details [here](https://docs.idena.io/docs/iip/iip-11).

**Comment by Zen-44** [comment]
Why are Suspended and Zombie identities eliminated from the calculation of the median_top100? 
This lowers the value since other identities with lower stakes are chosen instead to be in the top 100.

**Comment by midenaio** [comment]
@Zen-44 Suspended and Zombie do not participate in mining so similarly their stakes are not taken into account for `median_top100`

**Comment by illusiontorment** [comment]
I like this proposal. It will create a very interesting mix of PoS and PoP regarding the Governance of the network.

**Comment by teavar** [comment]
Why does it have to be based on stake amount? Coins can be bought, and in theory a few whales can collude to manipulate the network. How about using a combination of other indicators, e.g. Age + Score + Reported flips (within N number of validation ceremonies)? Or perhaps combining both: stake amount threshold + non financial indicators?

**Comment by ubiubi18** [comment]
@midenaio I get it why we have pool discrimination, but shouldnt after your iip-11 paradigm change surpassing the minimal stake threshold also overwrite delegates voting discrimination regarding verifying blocks - and make attacks even more expensive that way? These days we have many small pools with two to five honest high stake delegates, that suffer no matter than from pool discrimination - without surplus for network security?
